thunder/core/jit_ext.py

```diff
@@ -829,6 +829,51 @@ def _general_jit_bool_lookaside(wrapped_x: Any) -> bool | INTERPRETER_SIGNALS:
 
 _general_jit_lookaside_map[bool] = _general_jit_bool_lookaside
 
+
+def _torch_module_named_member_lookaside(model, get_member_func, prefix="", recurse=True, remove_duplicate=True):
```

Comment:
This is not a `_lookaside` but an `_impl`, maybe even with adding a comment that it will be interpreted.